public/js/sign_in.js

```diff
@@ -0,0 +1,32 @@
+
+const email = document.getElementById('email');
+const password = document.getElementById('password');
+const but = document.getElementById('butSend');
```

Comment:
what does but mean?
Variable names should be descriptive.

Reply:
for html class and id names you shouldn't use camelcase.
id should be ```but-send```

The name should be descriptive so I suggest you rename it to ```send-button```